cargo-creusot/src/main.rs

```diff
@@ -129,8 +128,8 @@ fn invoke_cargo(args: &CreusotArgs) {
     let mut cmd = Command::new(cargo_path);
     cmd.arg(format!("+{toolchain}"))
         .arg(&cargo_cmd)
-        .args(args.rust_flags.clone())
-        .env("RUSTC_WRAPPER", creusot_rustc_path)
```

Comment:
why the change from `RUSTC_WRAPPER` to `RUSTC`? (I have no idea what these variables do)

Reply:
`RUSTC=foo` replaces `rustc` with `foo`.
`RUSTC_WRAPPER=foo`  replaces `rustc` with `foo /path/to/rustc`, presumably for tools that just end up calling the underlying `rustc`. We abused this option to disambiguate when `cargo` called `creusot-rustc`.